### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Mihail-sudo**
Вот пока так, после этого с этого момента буду делать, меньше но чаще

### CODE REVIEW

File: nndt/space2/method_set.py [hunk 1]
```diff
@@ -52,6 +52,14 @@ def __init__(self, parent: AbstractBBoxNode = None):
 
     @node_method("sampling_grid(spacing=(D,H,W)) -> ns_xyz[D,H,W,3]")
     def sampling_grid(self, spacing: (int, int, int) = (2, 2, 2)) -> jnp.ndarray:
+        """Creates normilized space with uniform grid.
```
Comment (KonstantinUshenin) [inline nndt/space2/method_set.py:55]: normAlized

File: nndt/space2/transformation.py [hunk 2]
```diff
@@ -76,18 +76,50 @@ def __init__(self, ps_bbox: ((float, float, float), (float, float, float)),
 
     @node_method("transform_xyz_ps2ns(ps_xyz[...,3]) -> ns_xyz[...,3]")
     def transform_xyz_ps2ns(self, xyz: Union[onp.ndarray, jnp.ndarray]) -> Union[onp.ndarray, jnp.ndarray]:
+        """Transforms xyz phisical sapce to normilized space.
```
Comment (KonstantinUshenin) [inline nndt/space2/transformation.py:79]: physical space

**Review by KonstantinUshenin — CHANGES_REQUESTED** [review]
Quality of the docs are lower than I expect. English MUST be improved before merge. I also notice that some functions are wrongly annotated.

File: nndt/space2/method_set.py [hunk 3]
```diff
@@ -61,6 +69,16 @@ def sampling_grid(self, spacing: (int, int, int) = (2, 2, 2)) -> jnp.ndarray:
     @node_method("sampling_grid_with_noise(key, spacing=(D,H,W), sigma) -> ns_xyz[N,3]")
     def sampling_grid_with_noise(self, rng_key: PRNGKeyArray, spacing: (int, int, int) = (2, 2, 2),
                                  sigma=0.1) -> jnp.ndarray:
+        """Creates normilized space with noised grid.
```
Comment (KonstantinUshenin) [inline nndt/space2/method_set.py:72]: It is not a space creation method. All `sampling_` method is a data generators.
